### DISCUSSION TITLE
How to run [Explicit] tests from `dotnet test` in CI?

### DISCUSSION THREAD

**Discussion by Strepto** [nunit/nunit3-vs-adapter#959] [Q&A]
I have been googling, but have not found an answer for this.

I have a test suite with a number of tests that are marked with the `[Explicit]` Attribute.

When I run `dotnet test --project **/*.Tests.csproj` in my CI pipleine this does not run explicit tests (as I would expect).

Is there any argument to include explicit tests in a Command Line run that I have missed?

Thank you!

Using NUnit 3.12, NUnitTestAdapter 4.2.1

**Comment by OsirisTerje** [comment]
Earlier we had an automatic category called Explicit, but it doesn't work as it should after the latest changes.  We have also talked about adding a specific keyword, e.g. Explicit, that would sort of do what you propose.  What tends to be problems around these things, are when such keywords are being used together with other filters, but I guess we could add one and in the first round just let it work independently, and then fix the combinations later on.  I'll see if we can get this in the next version.   The current workaround would be to manually add a category to these tests that could be used as a filter.
